controllers/group_controller.go

```diff
@@ -67,12 +60,31 @@ func (ctrl *groupController) Create(context *gin.Context) {
 		return
 	}
 
+	// Make sure that every username was found in the database.
+	if len(users) != len(body.Usernames) {
+		context.AbortWithStatusJSON(http.StatusBadRequest, gin.H{"error": "some usernames were not found."})
+		return
+	}
+
 	//----------------------------------------------------------------------------------------
 
 	// Step 2: Find the id of each of the User models found.
-	var userIDs []primitive.ObjectID = make([]primitive.ObjectID, len(users))
+	// While iterating, make sure that the user who created the group is inside the userIDs.
+	userIDs := make([]primitive.ObjectID, len(users))
+	bearerUserFound := false
+
 	for i := 0; i < len(userIDs); i++ {
 		userIDs[i] = users[i].DefaultModel.IDField.ID
+
+		if !bearerUserFound && userIDs[i].Hex() == context.MustGet("userID") {
+			bearerUserFound = true
+		}
+	}
+
+	// Make sure that the user who created the group is inside the usernames.
+	if !bearerUserFound {
+		context.AbortWithStatusJSON(http.StatusBadRequest, gin.H{"error": "usernames must contain the bearer user"})
```

Comment:
Just a personal preference but would be nice to have standardized error messages (at least in terms of capitalizing first letter, punctuation at the end).